### PR TITLE
Feature/tozpatron 54 taby na glownym ekranie

### PR DESCRIPTION
**Pull request by michalsiedlecki**

### CODE REVIEW

**Comment by Bartosz-Kozajda** [comment]
All checks have failed. Please fix this.

File: app/src/main/java/com/intive/toz/MainActivity.java [hunk 1]
```diff
@@ -1,17 +1,79 @@
...
+    @NonNull
+    @Override
+    public NavigationTabsPresenter createPresenter() {
+        return new NavigationTabsPresenter();
+    }
+
+    private void initViews() {
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        viewPager = (ViewPager) findViewById(R.id.viewpager);
```
Comment (Bartosz-Kozajda) [inline app/src/main/java/com/intive/toz/MainActivity.java]: maybe try to use [Butterknife](http://jakewharton.github.io/butterknife/) to bind views

File: app/src/main/java/com/intive/toz/common/view/navigationTabs/ViewPagerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public int getCount() {
+        return mFragmentList.size();
+    }
+
+    public void addFragment(Fragment fragment, String title) {
+        mFragmentList.add(fragment);
+        mFragmentTitleList.add(title);
```
Comment (Bartosz-Kozajda) [inline app/src/main/java/com/intive/toz/common/view/navigationTabs/ViewPagerAdapter.java]: http://jakewharton.com/just-say-no-to-hungarian-notation/ 

:)

File: app/src/main/java/com/intive/toz/common/view/navigationTabs/Tab.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ * Navigation Tabs model class.
+ */
+
+public class Tab {
+
+    private int tabTitle;
+    private int tabIcon;
+
+    /**
+     *
```
Comment (KrzysztofPawlak) [inline app/src/main/java/com/intive/toz/common/view/navigationTabs/Tab.java]: unnecessary * line

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -9,7 +9,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/MyTheme.Base">
```
Comment (KrzysztofPawlak) [inline app/src/main/AndroidManifest.xml]: maybe AppTheme.Base

File: app/src/main/java/com/intive/toz/MainActivity.java [hunk 5]
```diff
@@ -47,17 +65,41 @@ public void onResponse(final Call<List<Pet>> call, final Response<List<Pet>> res
...
+        tabLayout = (TabLayout) findViewById(R.id.tabs);
+        tabLayout.setupWithViewPager(viewPager);
+        adapter = new ViewPagerAdapter(getSupportFragmentManager());
+    }
+
+
```
Comment (KrzysztofPawlak) [inline app/src/main/java/com/intive/toz/MainActivity.java]: blank line

**Pull request closed without merging**